finetune_adapter.py

```diff
@@ -53,6 +69,8 @@ def main():
     config = LLaMAConfig()
     config.block_size = block_size
 
+    # If you get an error here, check that you have downloaded the weights by following the
+    # instructions in the README.
```

Comment:
Maybe assert `path.isfile()` and point to the README if it fails

Reply:
Okay I added the error message. But I suggest we only make an exception here. In general let's keep the validation light in favor of minimalism.